### DISCUSSION TITLE
Tolgee self-hosting: problem with displaying accented characters in the machine translation result

### DISCUSSION THREAD

**Discussion by kanank** [tolgee/tolgee-platform#2379] [Q&A] [answered]
Hello.
Auto-translation inserts the correct line, but machine translation options are displayed (and inserted) without accented characters. It looks like encoding into the current system encoding, which does not contain these characters.
Tolgee works on Windows 10.

![image](https://github.com/tolgee/tolgee-platform/assets/17914922/d20b36bb-4b67-4350-b9b8-1fd0ace3d2a4)

**Comment by stepan662** [comment]
Hey, could you please give us more info? What version of Tolgee are you self-hosting, what browser are you using and also, could you try to run this command, in your browser console on the tolgee page:
```
document.characterSet
```
It should return "UTF-8", if the correct encoding is used.

**Comment by kanank** [comment, marked as answer]
Excuse me, the problem was not Tolgee, but JDK17. I have updated the JDK to version 22 - the problem has disappeared.
Thank you.
